Sources/GRPCCore/Streaming/Internal/RPCAsyncSequence+Buffered.swift

```diff
@@ -30,7 +30,7 @@ extension RPCAsyncSequence where Failure == any Error {
   }
 
   @inlinable
-  public static func _makeBackpressuredStream(
+  package static func _makeBackpressuredStream(
```

Comment:
We can remove the underscore too

Reply:
Oops, yeah we can actually get rid of the whole definition here :D